include/effects.h

```diff
@@ -2072,13 +2143,13 @@ typedef union {
     struct Effect86FXData*              unk_86;
 } EffectData;
 
-typedef struct EffectInstance {
+struct EffectInstance {
```

Comment:
this should still be the way it beed before (I think)

Reply:
Since now we're declaring `EffectInstance` at the top of the file, the compiler complains if we try to redefine the type here, so I've removed the typedef.